Cargo.toml

```diff
@@ -25,6 +25,7 @@ members = [
     "common/proxy",
     "common/rate-limiter",
     "common/retrier",
+    "common/security",
```

Comment:
`security` is a poorly named folder, it's for TCB stuff : o we actually have a `common/infallible` repo for APIs like this which I think would be a better place